lms/djangoapps/grades/tests/test_api.py

```diff
@@ -108,3 +112,126 @@ def test_override_subsection_grade(self, earned_graded):
             else:
                 assert history_entry.history_user is None
                 assert history_entry.history_user_id is None
+
+
+class ClearGradeTests(ModuleStoreTestCase):
+    """
+    Tests for the clearing grades api call
+    """
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.user = UserFactory()
+        cls.overriding_user = UserFactory()
+
+    @classmethod
+    def tearDownClass(cls):
+        super().tearDownClass()
+
+    def setUp(self):
+        super().setUp()
+        self.course = CourseFactory.create(org='edX', number='DemoX', display_name='Demo_Course', run='Spring2019')
```

Comment:
You don't need to provide these fields, you know, unless we care about their specific values. You can just let the factory create arbitrary values

Reply:
(Nothing wrong with providing them, but it implies to a reader that they're important, which in this case, they don't really seem to be)